src/paper/5-activity-risk.ts

```diff
@@ -74,7 +74,7 @@ Hopefully this puts the risk of indoor gatherings into perspective. To compare t
 Indoor unmasked interactions are quite risky, but being outdoors and wearing masks both make a _huge_ difference.
 
 
-**Masks**: We estimate that masks reduce your risk by 8x. This is if _both_ people are wearing masks. _Your_ mask decreases the risk to _you_ by about 2x. And _their_ mask decreases risk to _you_ by 4x. This assumes [a reasonably well-fitting surgical mask](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/). For more protection, there are other types of masks you can wear (addressed in the [Q&A](/paper/13-q-and-a)). Bandanas, buffs, or other single-layer coverings provide significantly less protection than we estimate here (see [Research Sources](/paper/14-research-sources), so for simplicity we treat them as "no mask", even though we do believe they provide some benefit.
+**Masks**: We estimate that masks reduce your risk by 8x. This is if _both_ people are wearing masks. _Your_ mask decreases the risk to _you_ by about 2x. And _their_ mask decreases risk to _you_ by 4x. This assumes [a reasonably well-fitting surgical mask](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/). For more protection, there are other types of masks you can wear (addressed in the [Q&A](/paper/13-q-and-a)). Bandanas, buffs, or other single-layer coverings provide significantly less protection than we estimate here (see [Research Sources](/paper/14-research-sources), so for simplicity we treat them as "no mask", even though we do believe they provide some benefit).
```

Comment:
I'm pretty sure the close paren is meant to be right after "Research Sources", not at the end of the sentence.

Reply:
Done